### ISSUE TITLE
Vale notifies for issues in AsciiDoc image lines

### ISSUE THREAD

**Issue by emteelb** [jdkato/txtlint#531]
Vale notifies for style rule errors in AsciiDoc `image:` lines. I don't see "image" as a [listed scope](https://vale.sh/docs/topics/scoping/#markup). Otherwise I would configure it either as an ignored or skipped scope.

Is it possible to implement?

Here's a contrived example:

```
$ cat test.adoc
image:centre.png[picture of a shopping centre,align='center',pdfwidth='75%']

$ vale test.adoc

 test.adoc
 1:40  error  Use 'center' rather than        RedHat.TermsErrors 
              'centre'.                                          

✖ 1 error, 0 warnings and 0 suggestions in 1 file.
```

**Comment by emteelb** [comment]
When thinking about this a bit more, it seems like behavior is actually all right. The first instance of "centre" is ignored, and the occurrence in the "user facing" content of the line "picture of a shopping centre" is flagged.

OK. However, if the occurrence is enclosed in backticks, that would otherwise be a skipped scope (because of configuration in my `vale.ini` file (?), it will get flagged:

```
$ cat test.adoc 
image:centre.png[enter a `centre` command,align='center',pdfwidth='75%']

enter a `centre` command

$ vale test.adoc

 test.adoc
 1:27  error  Use 'center' rather than        RedHat.TermsErrors 
              'centre'.                                          

✖ 1 error, 0 warnings and 0 suggestions in 1 file.
```

**Comment by emteelb** [comment]
Well, maybe I am confusing syntax here in AsciiDoc. If I change the file name to test.md, Vale reports no errors.

I do think my AsciiDoc syntax is correct for inline source code. So I'm not sure what the difference is for Vale between the `.md` and `.adoc` example files that I used.

**Comment by jdkato** [comment]
The relevant scope here is `alt`:

```ini
SkippedScopes = script, style, pre, figure, alt
```

Changing the file extension doesn't really make sense here -- the generated HTML will *not* be same. The (approximately) equivalent example in Markdown would be

```markdown
![enter a `centre` command](centre.png)
```

which has the same behavior since nested markup isn't supported in alt text (note: this is not specific to Vale).

**Comment by emteelb** [comment]
Thanks for the explanation and mentioning nested markup. Things make more sense to me now.
